cognite/client/_api_client.py

```diff
@@ -355,6 +356,7 @@ def _retrieve_multiple(
         params: dict[str, Any] | None = None,
         executor: ThreadPoolExecutor | None = None,
         api_subversion: str | None = None,
+        settings_forcing_raw_response_loading: list[str] | None = None,
```

Comment:
Remember to update the overload methods with the new parameter :)

Reply:
I don’t think that’s necessary when it doesn’t affect the return type?